### ISSUE TITLE
layer effects

### ISSUE THREAD

**Issue by circular17** [bgrabitmap/lazpaint#69]
- drop shadow
- stroke around
- fast blur (box blur?)

**Comment by lainz** [comment]
This remembers me to Photoshop, where you can create a nice looking 3d button only using layer effects. In fact any kind of filters may apply, but it is somewhat slow, because every filter is calculated when you paint on the layer.

**Comment by circular17** [comment]
Indeed, that may be slow. So I would keep only things that are fast enough. The other solution is to generate additional layers which would be better in layer groups. If in a group it is easy to generate a shadow layer, then one can update it when they have changed the source layer. Same with generating a Phong effect.

**Comment by circular17** [comment]
I've added drop shadow as a Python script for now.

**Comment by circular17** [comment]
Added color overlay as script

**Comment by circular17** [comment]
Other layer effect: layer shadow
See https://github.com/bgrabitmap/lazpaint/issues/129

**Comment by circular17** [comment]
There is a discussion now about how to implement layer effects in OpenRaster. That may give ideas on how to implement them.
https://lists.freedesktop.org/archives/create/2020-May/thread.html
